VistaCoreLibs/VistaOGLExt/VistaGLSLShader.h

```diff
@@ -209,6 +215,7 @@ class VISTAOGLEXTAPI VistaGLSLShader
 	bool m_bIsReadyForUse;
 
 	GLuint m_uiProgram;
+    std::vector<GLuint> m_vecComputeShader;
 	std::vector<GLuint>	m_vecVertexShader;
 	std::vector<GLuint> m_vecTessControlShader;
 	std::vector<GLuint> m_vecTessEvalShader;
```

Comment:
Why are all the shaders stored in vectors? Isn't there at most one of each in the pipeline?

Reply:
Because VISTA does that before. I just used the same methodoly as before